responsive-test-utils/src/main/java/dev/responsive/kafka/api/CassandraClientStub.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2023 Responsive Computing, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dev.responsive.kafka.api;
+
+import com.datastax.oss.driver.api.core.cql.AsyncResultSet;
+import com.datastax.oss.driver.api.core.cql.PreparedStatement;
+import com.datastax.oss.driver.api.core.cql.ResultSet;
+import com.datastax.oss.driver.api.core.cql.SimpleStatement;
+import com.datastax.oss.driver.api.core.cql.Statement;
+import dev.responsive.db.CassandraClient;
+import dev.responsive.db.RemoteKeyValueSchema;
+import dev.responsive.db.RemoteWindowedSchema;
+import dev.responsive.kafka.config.ResponsiveConfig;
+import dev.responsive.kafka.store.SchemaType;
+import dev.responsive.utils.RemoteMonitor;
+import java.util.OptionalInt;
+import java.util.Properties;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.ScheduledExecutorService;
+
+public class CassandraClientStub extends CassandraClient {
+
+  private final TTDKeyValueSchema kvSchema;
+  private final TTDWindowedSchema windowedSchema;
+
+  public CassandraClientStub(final Properties props) {
+    super(new ResponsiveConfig(props));
+    kvSchema = new TTDKeyValueSchema(this);
+    windowedSchema = new TTDWindowedSchema(this);
+  }
+
+  @Override
+  public ResultSet execute(final Statement<?> statement) {
+    return null;
```

Comment:
throughout this PR there's a pattern of returning `null` for all Cassandra related APIs, I'm wondering if it might make sense to instead return stubs that just throw exceptions if any of their methods are used? we can save that for a follow-up if this ever causes issues, but it makes me a bit nervous 😅

Reply:
That's fair, this was another instance of the philosophy of not going above and beyond unnecessarily at this point but I completely agree -- I put in a TODO but feel free to file a ticket